samples/AdvancedPushNotifications/src/main/java/com/swrve/sdk/demo/CustomLayoutSwrveGcmIntentService.java

```diff
@@ -0,0 +1,67 @@
+package com.swrve.sdk.demo;
+
+import android.app.Notification;
+import android.app.PendingIntent;
+import android.content.ContentResolver;
+import android.media.RingtoneManager;
+import android.net.Uri;
+import android.os.Bundle;
+import android.support.v4.app.NotificationCompat;
+import android.widget.RemoteViews;
+
+import com.swrve.sdk.SwrveHelper;
+import com.swrve.sdk.gcm.SwrveGcmIntentService;
+
+/*
+ * Use this class when you want to create richer local notifications form a push notification
+ * using a custom layout.
+ * http://developer.android.com/guide/topics/ui/notifiers/notifications.html
+ */
+public class CustomLayoutSwrveGcmIntentService extends SwrveGcmIntentService {
```

Comment:
Not sure we need this? The BigNotificationSwrveGcmIntentService already shows how to override the createNotification. That should be enough to demonstrate people can override this method and return whatever type of notification they like.